Makefile

```diff
@@ -209,8 +209,10 @@ validate: build-container
 # This target is only intended for development, e.g. executing it from an IDE. Use (make test) for CI or pre-release testing.
 test-all-local: validate-local test-unit-local
 
+.PHONY: validate-local
```

Comment:
OK. I guess failing to update the `.PHONY` at the top is good enough evidence that adding it directly above the target is more likely to cause future rule additions to notice. :)